### ISSUE TITLE
1.10.2 Ticking Entity

### ISSUE THREAD

**Issue by Thenosis** [WayofTime/BloodMagic#1114]
#### Issue Description: Ticking Entity [https://pastebin.com/riKetCEn](https://pastebin.com/riKetCEn)
I am not sure what caused this as I am not able to reproduce the problem more than once. I may have to try a fresh install of my mod pack as it may only happen when it is the first time a bound item is activated. I will try to test this more than once. I'm including a link to my modpack [R'lyehian Void](https://minecraft.curseforge.com/projects/rlyehian-void) so that you may test it within the confines of my custom pack. I'll bump this post once I've reloaded a completely fresh install of the pack. If you need any other info beyond what I've provided I'll be happy to accommodate to the best of my ability.

#### What happens:
 When I shift+right clicked to activate my bound shovel for the first time since creating a bound item the game crashed.

#### What you expected to happen: 
For the bound shovel to become activated.

#### Steps to reproduce:

1. Create a bound item (ie shovel, sword, etc)
2. Shift + Right click to activate

____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: BloodMagic-1.10.2-2.0.1-17.jar
- Minecraft: 1.10.2
- Forge: 12.18.3.2254

**Comment by TehNut** [comment]
>`BloodMagic-1.10.2-2.0.1-17.jar`

That jar doesn't exist.

`Build 17` was for 1.8.9. `2.0.1` didn't start until `Build 40` and was for 1.9.4.

According to your crash log, you're using `BloodMagic-1.10.2-2.1.7-76.jar`. Which is a much more sane version.

**Comment by Thenosis** [comment]
I know exactly what I did and why I typed in the wrong version. For some reason my eyes went to the blood arsenal version right above the Blood Magic one on my project Curse Voice page.

Sorry about that. I'll just copy and paste from the log next time. And speaking of saneness (is that a word) my pack IS about AbysallCraft. Cthulhu R'lyeh? Oh and I'll close this once I've done a fresh install and tested the update I see.
